programs/ambassadors/ambassadors.json

```diff
@@ -1,341 +1,341 @@
+        "date": {
+          "year": 2020,
+          "month": "August"
         },
-        {
-          "type": "video",
-          "title": "Supercharge your Angular Project with JSON Schema Forms",
-          "date": {
-            "year": 2020,
-            "month": "August"
-          },
-          "link": "https://www.youtube.com/watch?v=Xk9dxbbBFjo"
+        "link": "https://medium.com/@dashjoin/json-schema-schema-org-json-ld-whats-the-difference-e30d7315686a"
+      },
+      {
+        "type": "video",
+        "title": "Supercharge your Angular Project with JSON Schema Forms",
+        "date": {
+          "year": 2020,
+          "month": "August"
         },
-        {
-          "type": "adopter",
-          "title": "Add Dashjoin to the JSON Schema Adopters list.",
-          "date": {
-            "year": 2024,
-            "month": "April"
-          },
-          "link": "https://landscape.json-schema.org/"
-        }
-      ]
-    },      
-    {
-      "name": "David Biesack",
-      "img": "https://avatars.githubusercontent.com/u/545944?s=400&u=26818946106e2d8ae8c0cb5e0b72d6c7f612d268&v=4",
-        "bio": "Chief API Officer at APiture (I design banking APIs with OpenAPI 3.1, JSON Schema 2020/12) and author of the API Design Matters blog, https://apidesignmatters.substack.com/ ",
-      "title": "Chief API Officer",
-      "github": "DavidBiesack",
-      "mastodon": "@DavidBiesack@fosstodon.org",
-      "linkedin": "davdbiesack",
-      "company": "Apiture",
-      "country": "🇺🇸",
-      "contributions": [
-        {
-          "type": "article",
-          "title": "Composing API Models with JSON Schema",
-          "date": {
-            "year": 2023,
-            "month": "June"
-          },
-          "link": "https://apidesignmatters.substack.com/p/composing-api-models-with-json-schema"
+        "link": "https://www.youtube.com/watch?v=Xk9dxbbBFjo"
+      },
+      {
+        "type": "adopter",
+        "title": "Add Dashjoin to the JSON Schema Adopters list.",
+        "date": {
+          "year": 2024,
+          "month": "April"
         },
-        {
-          "type": "article",
-          "title": "Master JSON Schema's Subtleties",
-          "date": {
-            "year": 2023,
-            "month": "July"
-          },
-          "link": "https://apidesignmatters.substack.com/p/master-json-schemas-subtleties"
+        "link": "https://landscape.json-schema.org/"
+      }
+    ]
+  },      
+  {
+    "name": "David Biesack",
+    "img": "https://avatars.githubusercontent.com/u/545944?s=400&u=26818946106e2d8ae8c0cb5e0b72d6c7f612d268&v=4",
+      "bio": "Chief API Officer at APiture (I design banking APIs with OpenAPI 3.1, JSON Schema 2020/12) and author of the API Design Matters blog, https://apidesignmatters.substack.com/ ",
+    "title": "Chief API Officer",
+    "github": "DavidBiesack",
+    "mastodon": "@DavidBiesack@fosstodon.org",
+    "linkedin": "davdbiesack",
```

Comment:
@benjagm Please correct this line as well, this should be

`"linkedin": "davidbiesack",`

thanks!

Reply:
Thanks David! I just pushed the suggested changes